package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "lint": "eslint .",
     "prepare": "ncc build index.js -o dist/main --source-map --license licenses.txt",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --coverage --testTimeout=300000 --verbose",
+    "test": "mocha test/**/*.test.js",
```

Comment:
Instead of changing the test tool, just add your test to the existing test file https://github.com/open-turo/action-setup-tools/blob/main/node.test.js